source/content/guides/secrets/04-use-cases.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Pantheon Secrets Guide
+subtitle: Use Cases
+description: Some common uses cases for Pantheon Secrets
+terminuspage: true
+contributors: [stovak]
+contenttype: [guide]
+innav: [true]
+categories: [secrets]
+cms: [drupal, wordpress]
+audience: [development]
+product: [secrets]
+integration: [--]
+tags: [reference, cli, local, terminus, workflow]
+permalink: docs/guides/secrets/use-cases
+reviewed: "2024-05-01"
+---
+
+# Use Case: Using secrets with Integrated Composer
+
+## Mechanism 1: Oauth composer authentication (recommended)
+
+#### GitHub Repository
+
+1. [Generate a Github token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). The Github token must have all "repo" permissions selected.
+
+    NOTE: Check the repo box that selects all child boxes. **Do not** check all child boxes individually as this does not set the correct permissions.
+
+    ![image](https://user-images.githubusercontent.com/87093053/191616923-67732035-08aa-41c3-9a69-4d954ca02560.png) 
+
+1. Set the secret value to the token via terminus: `terminus secret:site:set <site> github-oauth.github.com <github_token> --type=composer --scope=user,ic`
```

Comment:
It would be better if the Terminus command was a block comment, like the other examples, instead of being an inline comment. Otherwise, it is hard to find when scanning the page.

We probably shouldn't include the "user" scope in examples.

Reply:
https://github.com/pantheon-systems/documentation/pull/8975/commits/134dce894236acac146d001ae55318dbbabef568